### ISSUE TITLE
Enabling Ad/Tracker doesn't load coverimage but still links to web article

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#3303]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Enabling Ad/Tracker doesn't load coverimage but still links to web article. iframe not blocked when adblock enabled creates a empty block on pages splitting the site header and the main content. 

**Expected behavior:**
Enabling Ad/Tracker should block the link to the webpage article and load the cover page. Works as expected on Chrome/FF/Opera. iframe containing ads blocked on Chrome/FF/Opera giving it a seamless look
- Platform (Win7, 8, 10? macOS? Linux distro?):
  Windows 10 x64
- Brave Version:
  0.11.6 Dev Beta 1
- Steps to reproduce:
  1. Visit [Autotrader.com](http://www.autotrader.co.uk/)
  2. Disable Ad/Tracker shows the correct content. 
  3. Enable Ad/Tracker in shields. Under main content the image is blocked but still links to the page article. Should display the cover image which should not link any article( Standard behavior on other browsers with adblock enabled)
- Screenshot if needed:
  https://www.dropbox.com/s/2mvetrpatszhthf/Untitled.mp4?dl=0
- Any related issues:

**Comment by bsclifton** [comment]
Closing in favor of https://github.com/brave/brave-browser/issues/1023
